app/models/proxy.rb

```diff
@@ -471,6 +473,15 @@ def sandbox_host
     URI(sandbox_endpoint || set_sandbox_endpoint).host
   end
 
+  def update_domains
+    domains = {
+      staging_domain: self.class.extract_domain(sandbox_endpoint.presence || default_staging_endpoint),
```

Comment:
Tiny thing. No need to call `presence` again.

Reply:
When again? `sandbox_endpoint` can be an emptu string no?

Reply:
You are right. Sorry. I was thinking about refactoring `extract_domain` and left that part of the thought only inside my head. No need to change anything.